src/Unosquare.Labs.EmbedIO/System.Net/WebSocketException.cs

```diff
@@ -95,7 +61,32 @@ internal static string GetMessage(CloseStatusCode code)
         /// </value>
         public CloseStatusCode Code { get; }
 
-        #endregion
+        internal static string GetMessage(CloseStatusCode code)
+        {
+            switch (code)
+            {
+                case CloseStatusCode.ProtocolError:
+                    return "A WebSocket protocol error has occurred.";
+                case CloseStatusCode.UnsupportedData:
+                    return "Unsupported data has been received.";
+                case CloseStatusCode.Abnormal:
+                    return "An exception has occurred.";
+                case CloseStatusCode.InvalidData:
+                    return "Invalid data has been received.";
+                case CloseStatusCode.PolicyViolation:
+                    return "A policy violation has occurred.";
+                case CloseStatusCode.TooBig:
+                    return "A too big message has been received.";
+                case CloseStatusCode.MandatoryExtension:
+                    return "WebSocket client didn't receive expected extension(s).";
+                case CloseStatusCode.ServerError:
+                    return "WebSocket server got an internal error.";
+                case CloseStatusCode.TlsHandshakeFailure:
+                    return "An error has occurred during a TLS handshake.";
+                default:
+                    return string.Empty;
```

Comment:
"Unknown status"?

Reply:
Nop, valid close status should not return message.